src/components/AssetDownload.tsx

```diff
@@ -102,34 +60,79 @@ const AssetDownload: React.FC<IProps> = ({
   const Svg = svg
 
   return (
-    <Container shouldHide={shouldHide}>
-      <h4>{title}</h4>
-      <div>
-        {children && <ImageContainer>{children}</ImageContainer>}
+    <Container
+      display={{
+        base: shouldHide ? "none" : "flex",
+        lg: "flex",
+      }}
+      maxW="100%"
+      minW="170px"
+      flex="1 1 45%"
+      flexDirection="column"
+      justifyContent="space-between"
+      m="4"
+      p={0}
+      opacity={shouldHide ? 0 : 1}
+    >
+      <Heading
+        as="h4"
+        fontSize={{ base: "1rem", md: "1.25rem" }}
+        fontWeight="500"
+      >
+        {title}
+      </Heading>
+      <Box>
+        {children && (
+          <Flex
+            border="1px"
+            borderColor="white700"
+            w="100%"
+            p={8}
+            textAlign="center"
+            justify="center"
+          >
+            {children}
+          </Flex>
+        )}
         {!children && (
-          <ImageContainer>
+          <Center border="1px" borderColor="white700" p={8} w="100%">
             {Svg && <Svg alt={alt} />}
-            {image && <Image image={getImage(image)!} alt={alt} />}
-          </ImageContainer>
+            {image && (
+              <Img
+                as={GatsbyImage}
+                image={getImage(image)!}
+                alt={alt}
+                w="100%"
+                alignSelf="center"
+              />
+            )}
+          </Center>
         )}
         {artistName && (
-          <Caption>
-            <ArtistSubtitle>
-              <Emoji text=":artist_palette:" mr={`0.5em`} />
+          <Flex
+            mb={4}
+            border="1px"
+            borderTop="none"
+            borderColor="white700"
+            p="0.5rem 1rem"
+            borderRadius="0 0 4px 4px"
+          >
+            <Flex mr={2} fontSize="md" textColor="text300">
+              <Emoji text=":artist_palette:" mr={2} fontSize="2xl" />
               <Translation id="page-assets-download-artist" />
-            </ArtistSubtitle>
+            </Flex>
             {artistUrl && <Link to={artistUrl}>{artistName}</Link>}
-            {!artistUrl && <span>{artistName}</span>}
-          </Caption>
+            {!artistUrl && <Text m={0}>{artistName}</Text>}
+          </Flex>
         )}
-      </div>
-      <ButtonContainer>
+      </Box>
+      <Container mt={4} p={0} maxW="none">
```

Comment:
I think there is no reason to use a `Container` here since we don't have any special width we want to keep on the content.
```suggestion
      <Box mt={4} p={0}>
```

Reply:
true, that makes sense.